src/app/store/spinner/reducers/spinner.reducer.ts

```diff
@@ -19,7 +20,7 @@ export const spinnerState = createSelector(getSpinnerState, (state) => state.sho
 export const spinnerReducer = createReducer(
   initialSpinnerState,
   on(TechnicalRecordServiceActions.getByVIN, TestResultActions.fetchTestResults, TestResultActions.fetchTestResultsBySystemId, (state) => ({ ...state, showSpinner: true })),
-  on(TechnicalRecordServiceActions.getByVINFailure, TechnicalRecordServiceActions.getByVINSuccess,
+  on(TechnicalRecordServiceActions.getByVINFailure,
```

Comment:
Also the tests are broken for this reducer, since as things stand, the spec file says the test should fail but it doesn't. I've just sent over a message to Tom with a 'fix' for the spec file, hopefully it's done tomorrow

Reply:
refactored